### PR TITLE
60%  keyboard with ISSI3733RGB IC

### PR DESCRIPTION
**Pull request by latincompass**
## Description

60% RGB keyboard use IS31FL3733 IC like HHKB 

## Types of Changes

- [x] Core
- [x] Bugfix
- [x] New feature
- [x] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/latin60rgb/config.h [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+#define DIODE_DIRECTION ROW2COL
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 3
+
+/* disable these deprecated features by default */
+#ifndef LINK_TIME_OPTIMIZATION_ENABLE
+#    define NO_ACTION_MACRO
+#    define NO_ACTION_FUNCTION
+#endif
```
Comment (drashna) [inline keyboards/latin60rgb/config.h:54]:
```suggestion
#define NO_ACTION_MACRO
#define NO_ACTION_FUNCTION
```

File: keyboards/latin60rgb/rules.mk [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+RGBLIGHT_ENABLE = no           # Enable keyboard RGB underglow
+MIDI_ENABLE = no               # MIDI support
+BLUETOOTH_ENABLE = no          # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no              # Audio output on port C6
+FAUXCLICKY_ENABLE = no         # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no            # Enable support for HD44780 based LCDs
+RGB_MATRIX_ENABLE = IS31FL3733 # Use RGB matrix
+NO_USB_STARTUP_CHECK = no      # Disable initialization only when usb is plugged in
+
+LAYOUTS = 60_latin_hhkb
```
Comment (drashna) [inline keyboards/latin60rgb/rules.mk:37]:
```suggestion
BLUETOOTH_ENABLE = no          # Enable Bluetooth
AUDIO_ENABLE = no              # Audio output
RGB_MATRIX_ENABLE = yes        # Use RGB matrix
RGB_MATRIX_DRIVER = IS31FL3733 
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/latin60rgb/latin60rgb.c [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        { 218,  64 },  { 178,  64 },                                           { 112,  64 },                                                         {  46,  64 }, {  26,  64 }
+    }, {
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,    1,
+        1, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+           1, 1,          4,             1, 1
+    }
+};
+#endif
```
Comment (drashna) [inline keyboards/latin60rgb/latin60rgb.c:109]: This should be moved to the end of the file, since there are other rgb matrix related features

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/latin60rgb/latin60rgb.c (hunk 3 above)
Comment (latincompass) [reply keyboards/latin60rgb/latin60rgb.c]: Thanks a lot Sir, I have fix it

File: keyboards/latin60rgb/latin60rgb.c [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        { 224,  48 }, { 198,  64 }, { 202,  48 }, { 180,  48 }, { 164,  48 }, { 148,  48 }, { 132,  48 }, { 116,  48 }, { 100,  48 }, {  84,  48 }, {  68,  48 }, {  52,  48 }, {  20,  48 }, {  10,  48 },
+        { 218,  64 },  { 178,  64 },                                           { 112,  64 },                                                         {  46,  64 }, {  26,  64 }
+    }, {
+        1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+        1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4,    1,
+        1, 1, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 4, 1,
+           1, 1,          4,             1, 1
+    }
+
```
Comment (drashna) [inline keyboards/latin60rgb/latin60rgb.c:108]:
```suggestion
};

```

File: keyboards/latin60rgb/latin60rgb.c [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    suspend_wakeup_init_user();
+}
+
+__attribute__ ((weak))
+void rgb_matrix_indicators_user(void) {
+    if (host_keyboard_led_state().caps_lock) {
+        rgb_matrix_set_color(41, 0xFF, 0xFF, 0xFF);
+    }
+}
+    };
```
Comment (drashna) [inline keyboards/latin60rgb/latin60rgb.c:125]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/latin60rgb/info.json [hunk 6]
```diff
@@ -0,0 +1,73 @@
+{
+  "keyboard_name": "latin60rgb",
+  "url": "",
+  "maintainer": "latincompass",
+  "width": 14,
+  "height": 5,
+  "layouts": {
+    "LAYOUT": {
```
Comment (drashna) [inline keyboards/latin60rgb/info.json:8]: This needs to match the layout macro name used by the keyboard. 
```suggestion
    "LAYOUT_60_latin_hhkb": {
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/latin60rgb/latin60rgb.c [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+     // { 0, B_14, A_14, C_14 },
+     // { 0, H_15, G_15, I_15 },
```
Comment (tzarc) [inline keyboards/latin60rgb/latin60rgb.c:16]:
```suggestion
```

File: keyboards/latin60rgb/rules.mk [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (tzarc) [inline keyboards/latin60rgb/rules.mk:12]:
```suggestion
```

**Review by tzarc — CHANGES_REQUESTED** [review]

File: keyboards/latin60rgb/latin60rgb.h [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+    K40, K41, K42, K4A, K4B       \
+) { \
+    { K00, K01, K02, K03, K04, K05, K06, K07, K08, K09, K0A, K0B, K0C, K0D }, \
+    { K10, K11, K12, K13, K14, K15, K16, K17, K18, K19, K1A, K1B, K1C, K1D }, \
+    { K20, K21, K22, K23, K24, K25, K26, K27, K28, K29, K2A, K2B,      K2C }, \
+    { K30, K31, K32, K33, K34, K35, K36, K37, K38, K39, K3A, K3B, K3C, K3D }, \
+    { K40, K41, K42, K4A, K4B, XXX, XXX, XXX, XXX,  XXX, XXX, XXX, XXX, XXX  }  \
+}
+
+#define LAYOUT_HHKB LAYOUT_60_latin_hhkb
```
Comment (drashna) [inline keyboards/latin60rgb/latin60rgb.h:36]:
```suggestion
```

File: keyboards/latin60rgb/config.h [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+#define RGB_DI_PIN B7
+#ifdef RGB_DI_PIN
+#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 8
+#define RGBLIGHT_HUE_STEP 5
+#define RGBLIGHT_SAT_STEP 5
+#define RGBLIGHT_VAL_STEP 5
+#define RGBLIGHT_SLEEP
+#endif
```
Comment (drashna) [inline keyboards/latin60rgb/config.h:88]: Does this even support RGB Light?   Because it's disabled in the rules.mk file for the keyboard.

Comment (latincompass) [reply keyboards/latin60rgb/config.h]: Ihave delete this,TAHNKS

**Review by drashna — APPROVED** [review]

**Comment by latincompass** [comment]
Thank you God drashna.

**Pull request merged**